qutebrowser/misc/utilcmds.py

```diff
@@ -33,6 +33,7 @@
 from PyQt5.QtCore import QUrl
 # so it's available for :debug-pyeval
 from PyQt5.QtWidgets import QApplication  # pylint: disable=unused-import
+from PyQt5.QtWebEngineWidgets import QWebEngineProfile  # pylint: disable=unused-import
```

Comment:
You can't rely on QtWebEngine being available - but why do you need to import this here at all?

Reply:
Right, this is actually something I meant to mention but I forgot. So basically, the reason is as in the comment above for `QApplication`---I wanted it available in `debug-pyeval`. And the reason for that is that I used it in BDD tests to verify that along with the qutebrowser option being set, the `QWebEngineProfile` property is properly set too. See `test_spell_bdd.py`. Now, the question is: can I do it in a cleaner way? This is basically the only way I could think of to test that the spell check is actually working.

Reply:
You can do `__import__('QWebEngineProfile').defaultProfile()` etc., but it's kind of a hack... I'll add some comments on those tests.